pdc/apps/common/viewsets.py

```diff
@@ -200,7 +200,8 @@ def initial(self, request, *args, **kwargs):
             self._check_ordering_keys(request)
 
     def _check_ordering_keys(self, request):
-        ordering_keys = request.query_params.get('ordering')
+        # Allow ordering to be defined multiple times and also as comma separated list.
+        ordering_keys = ','.join(request.query_params.getlist('ordering'))
```

Comment:
Not sure what issue  does this patch fix, could you give an example?

Currently we can add like this "ordering=aaa,bbb,ccc,ddd..." in a url.